main.css

```diff
@@ -0,0 +1,86 @@
+html, body {
```

Comment:
https://github.com/test-redi-js1/redi_js_spring_2020/pull/6#discussion_r407918524

I don't see a strong argument for your structure. Having folders per project has a clear advantage of separation of concerns along domain boundaries.

Reply:
moved css and js... shared stuff goes in main.css or main.js else each page gets it's own ex buttons/buttons.css

Reply:
Thr idea was to follow the standard structure: https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/Dealing_with_files , https://html5boilerplate.com/
Again, since yesterday we agreed on stickig to that approach I believe we shall either do so (setting up that structure was also part of git homework) or discuss and all agree on another approach. Otherweis we will keep changing it with every new pull request.

Reply:
For me discrete folders that contain each assignment is easier to work with. Example you had logic regarding buttons in the main js file. Why does that logic need to be loaded on the homepage? I also moved the button assignments files so you don't have to do anything...

Reply:
I dont mind either way, but standard approach is with js/css folders. And this was the approach we decided to follow and communicate to students on Thursday. Should you have a strong opinion agains the standard way of organising files, please create a discussion in slack or teams, so all teachers are involved and updated.

Reply:
There is no standard approach for file organization one must look at the problem they are trying to solve and design a file organization system around that. Our challenge is to find an organization system that fits the homework are beginners are creating. Each week they are assigned a task. Like build X thing using X knowledge. Putting things is separate folders for me makes life easier they everything to do with each assignment is in that folder. 

I've volunteering my time to create this stuff if I'm unable to create my assignments how I prefer then they won't get created.